app/workers/statuses_count_writeback_worker.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class StatusesCountWritebackWorker
+  include Sidekiq::Worker
+
+  def perform(account_id)
+    account = Account.find(account_id)
+    account.update(statuses_count: account.statuses_count(force: true))
```

Comment:
If force is true, this query will be executed multiple times every 5 minutes.

```sql
SELECT COUNT(*) FROM "statuses" WHERE "statuses"."account_id" = $1
```

I am worried about this.

Reply:
I think redis-base counting may be diffarence from real numbers, 
so I want clear cache periodically.
I think it is best to clear cache before write to DB.
do you have any good idea?

Reply:
Maybe simply remove `force`? Actually I don't know the reason why it should have `force`.